src/app/charts/line-graph.component.ts

```diff
@@ -182,4 +178,69 @@ export class LineGraphComponent {
     return [slope, intercept, rSquare];
   }
 
+  private drawMinMax(): void {
+    if (self.min || self.max) {
+      // Prepare standard variables
+      var x1 = self.xRange[1];
+      var x2 = self.xRange[0];
+      var y = D3.max(self.yData)/2;
```

Comment:
There are a few spots in this diff where spacing doesn't match our javascript style conventions. Strongly consider setting up something like eslint in your editor to warn you of these types of style violations.